disperser/dataapi/v2/server_v2.go

```diff
@@ -294,8 +307,95 @@ func (s *ServerV2) Shutdown() error {
 	return nil
 }
 
+// FetchBlobFeedHandler godoc
+//
+//	@Summary	Fetch blob feed
+//	@Tags		Blob
+//	@Produce	json
+//	@Param		end					query		string	false	"Fetch blobs up to the end time (ISO 8601 format: 2006-01-02T15:04:05Z) [default: now]"
```

Comment:
This is formatted by swagger. It's perfectly aligned in editor locally, not sure why they appear misaligned here.